### ISSUE TITLE
mobilenetv2 pretrained

### ISSUE THREAD

**Issue by Tim4AI** [georgymironov/single_shot_detection#1]
hello, how to input the mobilenetv2 pretrained model.

**Comment by georgymironov** [comment]
Hello.

I used the weights imported from Keras. Those can be obtained using converter from the repo.
```
from bf.base import mobilenet_v2_10
model = mobilenet_v2_10()
model.init_from_keras()
torch.save(model.state_dict(), 'weights_mobilenet_v2_keras.pt')
```
When it's done add the path to config file
```
model = {
    'base': {
         'name': 'mobilenet_v2_10',
         'weight': './weights_mobilenet_v2_keras.pt'
    },
    ...
}
...
```

**Comment by Tim4AI** [comment]
thanks,i have solved this problem by
model = {
    'base': {
         'name': 'mobilenet_v2_10',
         'weight': 'keras'
    },
    ...
}
another question, i  change retina_rn50_500_voc.py config file use the way of up but failed.

**Comment by georgymironov** [comment]
`'weight': 'keras'` will work but it's gonna use more gpu memory this way as it creates tensorflow session.
As for retinanet, i have just pushed a fix for focal loss so you need to update to the latest commit.
Also there is one more line in config should be changed:
```
model = {
  'detector': {
    'features': {
      ...
      'out_layers': ([13, 'expand_relu'], 18),
      ...
    }
  }
}
```

**Comment by Tim4AI** [comment]
thanks, i have tested success.
